services/authoring/src/security/types.d.ts

```diff
@@ -0,0 +1,6 @@
+declare namespace Express {
+  export interface User {
```

Comment:
where's the usage for this?

Reply:
https://github.com/Soluto/tweek/blob/da6bc54ea7b8b2f30a656683714ad1b484551a03/services/authoring/src/security/authorize.ts#L14
https://github.com/Soluto/tweek/blob/da6bc54ea7b8b2f30a656683714ad1b484551a03/services/authoring/src/security/authorize.ts#L20-L21
https://github.com/Soluto/tweek/blob/da6bc54ea7b8b2f30a656683714ad1b484551a03/services/authoring/src/security/strategies/tweek-internal.ts#L6